src/gui/mzroll/background_peaks_update.cpp

```diff
@@ -587,11 +587,21 @@ void BackgroundPeakUpdate::align() {
                         // if jDoc is null that means the json returned from python is malformed
                         // in such a case our rts wont update with new values
                         jDoc = QJsonDocument::fromJson(processedDataFromPython);
+
+                        QString errorMessage=QString::number(processedDataFromPython.size());
+                        errorMessage="Received data: "+errorMessage+" bytes";
```

Comment:
It does not make sense to start forming the error message at this point. We don't know whether we will actually use it